runtime/ms_rest_azure/ms_rest_azure2.gemspec

```diff
@@ -7,21 +7,21 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'ms_rest_azure/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = 'ms_rest_azure'
+  spec.name          = 'ms_rest_azure2'
   spec.version       = MsRestAzure::VERSION
   spec.authors       = 'Microsoft Corporation'
   spec.email         = 'azsdkteam@microsoft.com'
 
   spec.summary       = %q{Azure Client Library for Ruby.}
   spec.description   = %q{Azure Client Library for Ruby.}
-  spec.homepage      = 'https://aka.ms/ms_rest_azure'
+  spec.homepage      = 'https://github.com/chef/azure-sdk-for-ruby'
```

Comment:
this one could be updated to `https://github.com/chef/azure-sdk-for-ruby/tree/master/runtime/ms_rest_azure`

Reply:
Done